lib/emque/consuming/adapters/rabbit_mq/worker.rb

```diff
@@ -31,19 +31,18 @@ def initialize(connection, topic)
               channel.prefetch(config.adapter.options[:prefetch])
             end
 
-            self.queue =
-              channel
-                .queue(
-                  "emque.#{config.app_name}.#{topic}",
-                  :durable => config.adapter.options[:durable],
-                  :auto_delete => config.adapter.options[:auto_delete],
-                  :arguments => {
-                    "x-dead-letter-exchange" => "#{config.app_name}.error"
-                  }
-                )
-                .bind(
-                  channel.fanout(topic, :durable => true, :auto_delete => false)
-                )
+            self.queue = channel
+              .queue(
+                "emque.#{config.app_name}.#{topic}",
+                :durable => config.adapter.options[:durable],
+                :auto_delete => config.adapter.options[:auto_delete],
+                :arguments => {
+                  "x-dead-letter-exchange" => "#{config.app_name}.error"
+                }
+              )
+              .bind(
+                channel.fanout(topic.to_s, :durable => true, :auto_delete => false)
```

Comment:
Looks like the only change here is sending topic as a string, might be more readable if we split this into multiple lines, otherwise looks good!

```suggestion
                channel.fanout(
                  topic.to_s, 
                  :durable => true, 
                  :auto_delete => false,
                 )
```